Segments/Blackburrow.mapproj

```diff
@@ -1298,13 +1298,66 @@
         }
     }
     
+public ItemQuality getRandomQuality()
+{
+    // This method will allow me to randomly generate an item quality for loot that isn't controlled by lootpacks
+    var chance = Utility.Random(1,1000);
+    ItemQuality quality = ItemQuality.Common;
+
+    if (chance < 500)
+    {
+        return quality;
+    }
+
+    if (chance >= 500 && chance < 750)
+    {
+        quality = ItemQuality.Uncommon;
```

Comment:
You probably want to set this to common and the local var to uncommon. Change < 500 being uncommon.